myapp/spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,243 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Task', type: :system do
+  before do
+    create(:user)
+  end
+
+  describe '#index' do
+    before do
+      visit root_path
+    end
+
+    it 'display default item' do
+      expect(page).to have_link 'Add Task'
+    end
+
+    it 'go to New Task page' do
+      click_on 'Add Task'
+      expect(current_path).to eq new_task_path
+    end
+
+    context 'when user has tasks' do
+      before do
+        create(:task)
+        visit current_path
+      end
+
+      it "return user's task" do # rubocop:disable RSpec/MultipleExpectations
+        expect(page).to have_link '散歩'
+        expect(page).to have_link 'Edit'
+        expect(page).to have_link 'Delete'
+      end
+    end
+
+  end
+
+  describe '#show' do
+    let(:task) { create(:task) }
+
+    before do
+      visit task_path(task)
+    end
+
+    it 'return task info' do # rubocop:disable RSpec/MultipleExpectations
+      expect(page).to have_content task.name
+      expect(page).to have_content task.description
+      expect(page).to have_content task.priority
+      expect(page).to have_content task.limit
+    end
+
+    context 'when user click Edit' do
+      it 'return edit task form' do
+        click_on 'Edit'
+        expect(page).to have_current_path edit_task_path(task), ignore_query: true
+      end
+    end
+
+    context 'when user click Delete' do
+      it 'delete task' do
+        expect {
+          click_on 'Delete'
+        }.to change { Task.count }.by(-1)
```

Comment:
ブロックではなく、第一引数にクラスと第二引数にメソッド（シンボル）を渡しても問題なく動きますよ:thumbsup:
https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ExpectChange

```suggestion
        }.to change(Task, :count).by(-1)
```

Reply:
ちなみにRubyで上記のような引数の渡し方や連想配列のkeyで文字列ではなくシンボル（`:symbol`）を使う事が多いのですが、以下の記事ような理由があったりしますー
https://zenn.dev/kanoe/articles/352d78902c83e168db66

ご存知でしたらご放念ください:pray:

Reply:
なるほど、シンボルを使う方が読み手的には好ましいということでしょうか。

Reply:
ですね、下記の都合を両立させるためのものですー

* 読み手的としては文字列みたいなものが読みやすい
* 計算機上は整数として扱ったほうが効率が良い